pkg/adapter/network.go

```diff
@@ -85,12 +86,47 @@ func (r *LocalRuntime) NetworkInspect(cli *cliconfig.NetworkInspectValues) error
 }
 
 // NetworkRemove deletes one or more CNI networks
-func (r *LocalRuntime) NetworkRemove(cli *cliconfig.NetworkRmValues) error {
+func (r *LocalRuntime) NetworkRemove(ctx context.Context, cli *cliconfig.NetworkRmValues) error {
+	containers, err := r.GetAllContainers()
+	if err != nil {
+		return err
+	}
 	for _, name := range cli.InputArgs {
+		// We need to iterate containers looking to see if they belong to the given network
+		for _, c := range containers {
+			if util.StringInSlice(name, c.Config().Networks) {
+				// if user passes force, we nuke containers
+				if cli.Force {
+					if err := r.RemoveContainer(ctx, c.Container, true, true); err != nil {
+						return errors.Wrapf(err, "error while removing network '%s'", name)
+					}
+				} else {
+					// Without the force option, we return an error
+					return errors.Errorf("unable to remove network '%s': it has one or more containers associated with it", name)
```

Comment:
Do we usually error out if one of the networks is blocked, should we continue on to the next?

Reply:
im torn on this ... @mheon does like it to try and complete.  on the other hand, if this fails, we really cannot complete the command

Reply:
lemme know what you guys want and I can handle it

Reply:
`podman rmi image1 image2 image3`
Will delete all images that it can, and report images that it can not because of containers.
I see no reason why 
podman network rm net1 net2 net3
Shold not do the same.

Reply:
reworked